contrib/PFAS_concentrations/requirements.txt

```diff
@@ -0,0 +1,8 @@
+git
+loguru
+openpyxl
+OWSLib
+packaging
+pandas
+tqdm
+pydov[proxy]>=3.0.0
```

Comment:
I think this needs to be 3.1.0? And do we (can we) have a similar version specification in the environment.yml?